demos/serverless/src/handlers.js

```diff
@@ -4,6 +4,7 @@
 const AWS = require('./aws-sdk');
 const fs = require('fs');
 const { v4: uuidv4 } = require('./uuid');
+const { metricScope } = require('aws-embedded-metrics');
```

Comment:
Import this library from a local path (`./`) since the serverless deployment copies dependencies into a local folder.